rscript/r-script-curl-aws

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:14.10
```

Comment:
If you set the `FROM` to `FROM mesosphere/rscript-curl` then you can avoid re-installing all the packages it has installed. This'll be MUCH quicker.